tuta-sdk/rust/sdk/src/entities/entity_facade.rs

```diff
@@ -41,15 +62,238 @@ pub trait EntityFacade: Send + Sync {
 		entity: ParsedEntity,
 		resolved_session_key: ResolvedSessionKey,
 	) -> Result<ParsedEntity, ApiCallError>;
+
+	fn encrypt_and_map_to_literal(
```

Comment:
I don't know if the "literal" in the name makes sense anymore, that was kind of js-speak for "object" I guess